test/phases/select_action_phase.js

```diff
@@ -63,4 +67,24 @@ describe('Select Action Phase', () => {
     expect(actions['Second action']).to.be.calledBefore(actions.Quit);
     expect(actions.Quit).to.be.calledOnceWith();
   });
+
+  it('displays correct dialog when account has insufficient funds to perform transaction', async () => {
```

Comment:
suggestion: you could wrap this two `it` cases into a `describe` with `in case of errors` with the first one saying `displays correct dialog when account has insufficient funds` and the second one `displays generic error dialog otherwise`